appveyor.yml

```diff
@@ -62,6 +62,8 @@ environment:
       solution_name: win_build\boinc_vs2013.sln
       depends_git_path: http://boinc.berkeley.edu/git/boinc_depends_win_vs2013.git
       depends_path: C:\projects\boinc_depends_win_vs2013
+  BINTRAY_API_KEY:
```

Comment:
Please also fix the indentation of this line to match the level above. The next line needs to get two more spaces so it is slightly more indented than the first line.

Reply:
@ChristianBeer, are you sure? I thought that BINTRAY_API_KEY should be under environment section, not matrix or matrix entry level....

Reply:
You are right. It's on the correct level.